actions/bumper/entrypoint.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh
+
+# based off https://github.com/anothrNick/github-tag-action
+
+set -eu
+
+# input configuration and defaults
+RELEASE_BRANCH_LIST=${INPUT_RELEASE_BRANCH_LIST:-develop,master}
+DEFAULT_SEMVER_BUMP=${INPUT_SEMVER_BUMP:-minor}
+VERSION_FILE_PATH=${INPUT_VERSION_FILE_PATH:-}
+INITIAL_VERSION=${INPUT_INITIAL_VERSION:-0.0.0}
+SEMVER_WITH_V=${INPUT_SEMVER_WITH_V:-false}
+
+# use comma as input field separator
+PREVIFS=${IFS}
+IFS=','
+
+# check whether we are on a prerelease branch
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+PRE_RELEASE=true
+for BRANCH in ${RELEASE_BRANCH_LIST} ; do
+    if expr "${CURRENT_BRANCH}" : "${BRANCH}" 1>/dev/null; then
+        PRE_RELEASE=false
+    fi
+done
+
+# restore previous IFS variable
+IFS=${PREVIFS}
+
+# get all commits since last semantic version
+CURRENT_SEMVER=$(git for-each-ref --sort=-v:refname --count=1 --format '%(refname)' refs/tags/[0-9]*.[0-9]*.[0-9]* refs/tags/v[0-9]*.[0-9]*.[0-9]* | cut -d / -f 3-)
+CURRENT_SEMVER=0.50.0
+if [ -z "${CURRENT_SEMVER}" ]; then
+    echo "No semantic versions. Starting from ${INITIAL_VERSION}"
+    GIT_LOG=$(git log --pretty='%B')
+    CURRENT_SEMVER="${INITIAL_VERSION}"
+    CURRENT_COMMIT=$(git show -s --format=%H)
+else
+    GIT_LOG=$(git log "${CURRENT_SEMVER}..HEAD" --pretty='%B')
+    SEMVER_COMMIT=$(git rev-list -n 1 "${CURRENT_SEMVER}")
+    CURRENT_COMMIT=$(git rev-parse HEAD)
+
+    if [ "${SEMVER_COMMIT}" = "${CURRENT_COMMIT}" ]; then
+        echo "No new commits since previous semantic version. Skipping semantic version bump."
+        exit 0
+    fi
+fi
+
+printf "### Commits since %s: ###\n\n%s\n\n### End of commits ###\n\n" "${CURRENT_SEMVER}" "${GIT_LOG}"
+
+# bump the semantic version using the default bump level or exit if default is "none"
+defaultbump() {
+    if [ "${DEFAULT_SEMVER_BUMP}" = "none" ]; then
```

Comment:
can `DEFAULT_SEMVER_BUMP` ever be `"none"`? It doesn't look like it, since it defaults to `minor`

Reply:
You can set `INPUT_SEMVER_BUMP="none"`, which would overwrite the default.

Reply:
That behavior isn't documented in the README or `usage`

Reply:
I added it to the usage description. Let me know if that's enough.